Source/SwiftLintFramework/Extensions/File+SwiftLint.swift

```diff
@@ -126,3 +126,17 @@ extension File {
         lines = contents.lines()
     }
 }
+
+extension File {
+    internal func ruleEnabledViolatingRanges(violatingRanges: [NSRange], forRule rule: Rule)
```

Comment:
This function looks good, but why is it in a separate extension? Please just move to the main extension.